.python-version

```diff
@@ -0,0 +1 @@
+3.13.0
```

Comment:
Can't this be part of pyproject.toml?

Reply:
It can. `uv` creates this by default, given some Python toolings read this file. I will remove this and keep it in `pyproject.toml`, then update `workflow` to revert to its original version selection.

Reply:
Addressed in [`6634c9b` (#579)](https://github.com/archlinux/archweb/pull/579/commits/6634c9b341b841ead2f39c372e3fe7cf32121fb7)